inotify.go

```diff
@@ -278,27 +275,6 @@ func (w *Watcher) readEvents() {
 	}
 }
 
-// Certain types of events can be "ignored" and not sent over the Events
-// channel. Such as events marked ignore by the kernel, or MODIFY events
-// against files that do not exist.
-func (e *Event) ignoreLinux(mask uint32) bool {
-	// Ignore anything the inotify API says to ignore
-	if mask&syscall.IN_IGNORED == syscall.IN_IGNORED {
```

Comment:
Ah, okay. Adding the flags to the event seems like a nice improvement. So we'd drop events with `IN_IGNORE` on our code instead of in fsnotify?. Also, do you think this is a change we'll be able to get upstream?